foundation/foundation-configuration-store/src/test/java/com/boozallen/aissemble/configuration/store/LoadConfigurationsSteps.java

```diff
@@ -161,4 +198,53 @@ private void assertPropertySetsEqual(Set<Property> expected, Set<Property> resul
             assertTrue("Could not find " + expectedProperty, matchFound);
         }
     }
-}
+
+    private class ConfigStoreInitMock extends ConfigStoreInit {
```

Comment:
Q: can we achieve this by leveraging the existing ConfigStoreInitTestHelper?

Reply:
an error would show `"LoadConfigurationsSteps.java:[80,24] init() has protected access in com.boozallen.aissemble.util.ConfigStoreInitTestHelper"` as the void function is protected

Reply:
You should be able to just use the `ConfigStoreInitTestHelper` class without creating a new test class to override the init() function of the `ConfigStoreInitTestHelper`.

Reply:
gotcha, changed it to using ConfigStoreInitTestHelper